core/types/block.go

```diff
@@ -79,7 +79,7 @@ func (n *BlockNonce) UnmarshalJSON(input []byte) error {
 	return nil
 }
 
-// Header represents Ethereum block headers.
+// Header represents a block header in the Ethereum blockchain.
 type Header struct {
```

Comment:
I made these so it's consistent with the block's doc. If you really are opposed I can change it back, though in general I don't see why polishing the docs is a bad thing. Even if they don't provide extra "meaning".